tests/test_source.py

```diff
@@ -42,6 +42,15 @@ def THREDDSMergedSource_cat_short(
     return cat
 
 
+@pytest.fixture(scope='module')
+@pytest.mark.parametrize(
+    'path', ['air.sig995.194*.nc', ['air.sig995.194*.nc'], pytest.param(1, marks=pytest.mark.xfail)]
+)
```

Comment:
How about putting test for errors in a separate test?


```python
@pytest.mark.parametrize('path', [1, [1, 'air.sig995.194*.nc']])
def THREDDSMergedSource_path_error(THREDDSMergedSource_cat_short_url, path):
   with pytest.raises(ValueError):
      intake.open_thredds_merged(THREDDSMergedSource_cat_short_url, path)
```



```suggestion
@pytest.mark.parametrize(
    'path', ['air.sig995.194*.nc', ['air.sig995.194*.nc']]
)
```